FBSnapshotTestCase/FBSnapshotTestCase.h

```diff
@@ -28,11 +24,13 @@
  */
 #define FBSnapshotVerifyViewWithOptions(view__, identifier__, suffixes__) \
 { \
+NSString *envReferenceImageDirectory = [NSProcessInfo processInfo].environment[@"FB_REFERENCE_IMAGE_DIR"]; \
```

Comment:
Yikes, I see. So to keep backwards compatibility with -D flags in ObjC, we'd get an incorrect -D flag value for Swift - but we'd encourage Swift users to use the environment variable. Can you think of any other ways to maintain backwards compatibility with -D flags in ObjC and support the env var?